config/webpack_helpers.js

```diff
@@ -372,6 +391,10 @@ function getViewerConfig(options) {
           target: 'web',
           plugins: [
             htmlPlugin,
+            new MiniCssExtractPlugin({
```

Comment:
I guess it is a no-op, but it seems like this should only be needed in the case of the module build.

Reply:
It also works for your main build target as well. It extracts the css into a separate file, which allows the browser to load it in parallel with the javascript. The end result is a faster initial load time for neuroglancer.

Reply:
Okay, if it improves loading time then changing it sounds good.  That is how it was originally (using ExtractTextPlugin instead of MiniCssExtractPlugin), but there was an incompatibility with the dynamic module loading of tfjs for some reason, so we changed to the current CSS embedded in JS method.  However, the Python integration (python/neuroglancer/static, python/setup.py) will also have to be modified to account for the added files.

Reply:
Does the modification require anything more than adding the css files to the static_files array?

Reply:
python/neuroglancer/static/__init__.py also needs to be modified (at the moment the list is duplicated there)

Reply:
I took care of that and also changed the .gitignore file in the static directory